presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/README.md

```diff
@@ -0,0 +1,148 @@
+# Presto planner and optimizer tests
+
+Testing planner and optimizer functionality generally requires generating a
```

Comment:
Keeping it here, depending on implementation, there may be other ways to test it. Open to other suggestions that cover that though.